client/src/related_items/RelatedProducts.jsx

```diff
@@ -23,9 +23,9 @@ const RelatedProducts = (props) => {
   }, [productId])
 
   return (
-    <div id="Related-Products" >
+    <div id="related-products" >
       <h1 id="related-product-header">Related Products</h1>
-      <div className="related-card">
+      <div id="related-cards">
         {relatedProducts.map(productId => (
```

Comment:
not sure about this, but can you omit props.relatedProducts?

Reply:
The reason Im able to do it, is that in this case, relatedProducts is state part of my state, Im not getting from props